mlir/tools/rocmlir-gen/rocmlir-gen.cpp

```diff
@@ -124,6 +124,11 @@ static llvm::cl::opt<int> num_cu(
                    "gfx906(60/64), gfx908(120)"),
     llvm::cl::value_desc("compute unit value"), llvm::cl::init(0));
 
+static llvm::cl::opt<int> numChiplets("num_chiplets",
```

Comment:
add a test with non-default num_chiplets to see if it is being applied correctly as the attribute or not.